test/fixtures/compound11.expected.css

```diff
@@ -0,0 +1,12 @@
+
+#foo a li {
+	color: #ffffff;
+}
+
```

Comment:
It seems like this rule is missing from the output.

 - In an ideal world, should this be in the output?
 - Is this one of the known limitations? That's fine, but let's add a comment about it

```css
#foo.bar a + li {
	color: #000000;
}
```